src/Razor/src/RazorDeployment/RazorDeployment.csproj

```diff
@@ -11,6 +11,7 @@
     <VSSDKTargetPlatformRegRootSuffix>RoslynDev</VSSDKTargetPlatformRegRootSuffix>
   </PropertyGroup>
   <ItemGroup>
+    <PackageReference Include="Microsoft.VisualStudio.Copilot" Version="$(MicrosoftVisualStudioCopilotVersion)" />
```

Comment:
These were needed due to a dependency on a version that isn't published where we have access. This specifies a version higher to satisfy that requirement. Hopefully we can remove at some point...

Reply:
Am I understanding correctly, you're saying one of the VS packages we bring in transitively references a version of copilot that isn't published, so we reference this to pin a version that is accessible?

What a wonderful mess :D

Reply:
Correct. It is a wonderful mess...